modules/dublincore/src/main/java/org/opencastproject/metadata/dublincore/DublinCoreUtil.java

```diff
@@ -102,15 +101,13 @@ public DublinCoreCatalog apply(MediaPackageElement mpe) {
    * @return the catalog
    */
   public static DublinCoreCatalog loadDublinCore(Workspace workspace, MediaPackageElement mpe) {
-    InputStream in = null;
-    try {
-      in = new FileInputStream(workspace.read(mpe.getURI()));
+    URI uri = mpe.getURI();
+    logger.debug("Loading DC catalog from {}", uri);
+    try (InputStream in = workspace.read(uri)) {
       return DublinCores.read(in);
     } catch (Exception e) {
-      logger.error("Unable to load metadata from catalog '{}': {}", mpe, e);
+      logger.error("Unable to load metadata from catalog '{}'", mpe, e);
       return chuck(e);
-    } finally {
```

Comment:
No need to close the input stream anymore?

Reply:
Java's try-with-resources statement automatically closes the `InputStream`.